### DISCUSSION TITLE
Webhook Body Template configuration

### DISCUSSION THREAD

**Discussion by powerscooter** [netbox-community/netbox#10620] [Q&A] [answered]
Hello everybody,

i have this output on my webhook server without any Body Template configuration in Netbox:

![image](https://user-images.githubusercontent.com/18429672/195200776-9736c7f4-5d83-42e5-ba52-4792a311241d.png)

I want the value: data > a_terminations > object > device > name

Can someone say me, how i have to configure the Body template for that?

I tried this way, but this is not working because of the nested json.

`{
"cable_label": "{{ data['display'] }}",
"a_device": "{{ data.a_terminations.object.device.name }}",
"tags": "{{ data['tags'][0]['name'] }}"
}`

Thanks to all!

**Comment by candlerb** [comment, marked as answer]
Netbox 3.3?

As you can see from your screenshot, a_terminations is a list, so you can use `a_terminations[0]` - although if the cable has multiple terminations at that end you'll only be looking at the first one.

**Reply by powerscooter** [reply]
Thank you this has worked
